package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "ts-to-zod",
+  "name": "ts-to-zod-plus",
   "description": "Generate zod schemas from typescript types/interfaces",
-  "version": "1.13.1",
+  "version": "1.0.1",
```

Comment:
Please revert this

Reply:
Thx for reviewing, I have reverted it! Because I was in a hurry to use so I publish another package.